colabs/Ladowanie_danych_do_pandasDF.ipynb

```diff
@@ -0,0 +1,487 @@
+              "<style scoped>\n",
+              "    .dataframe tbody tr th:only-of-type {\n",
+              "        vertical-align: middle;\n",
+              "    }\n",
+              "\n",
+              "    .dataframe tbody tr th {\n",
+              "        vertical-align: top;\n",
+              "    }\n",
+              "\n",
+              "    .dataframe thead th {\n",
+              "        text-align: right;\n",
+              "    }\n",
+              "</style>\n",
+              "<table border=\"1\" class=\"dataframe\">\n",
+              "  <thead>\n",
+              "    <tr style=\"text-align: right;\">\n",
+              "      <th></th>\n",
+              "      <th>nconst</th>\n",
+              "      <th>tconst</th>\n",
+              "      <th>titleType</th>\n",
+              "      <th>originalTitle</th>\n",
+              "      <th>isAdult</th>\n",
+              "      <th>startYear</th>\n",
+              "      <th>endYear</th>\n",
+              "      <th>genres</th>\n",
+              "      <th>category</th>\n",
+              "      <th>characters</th>\n",
+              "      <th>...</th>\n",
+              "      <th>year_globes</th>\n",
+              "      <th>category_globes</th>\n",
+              "      <th>film_globes</th>\n",
+              "      <th>win_globes</th>\n",
+              "      <th>year_emmy</th>\n",
+              "      <th>category_emmy</th>\n",
+              "      <th>nominee_emmy</th>\n",
+              "      <th>company_emmy</th>\n",
+              "      <th>producer_emmy</th>\n",
+              "      <th>win_emmy</th>\n",
+              "    </tr>\n",
+              "  </thead>\n",
+              "  <tbody>\n",
+              "    <tr>\n",
+              "      <th>0</th>\n",
+              "      <td>nm0000004</td>\n",
+              "      <td>[tt0694438, tt0694654, tt0694932, tt0694936, t...</td>\n",
+              "      <td>[tvEpisode, tvEpisode, tvEpisode, tvEpisode, t...</td>\n",
+              "      <td>[Jill Clayburgh/Leon Redbone/The Singing Idler...</td>\n",
+              "      <td>[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ...</td>\n",
+              "      <td>1973</td>\n",
+              "      <td>\\N</td>\n",
+              "      <td>[Comedy,Music, Comedy,Music, Comedy,Music, Com...</td>\n",
+              "      <td>actor</td>\n",
+              "      <td>[[\"Jawaharlal Nehru\",\"Various\"], [\"Larry Farbe...</td>\n",
+              "      <td>...</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "      <td>[]</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>1</th>\n",
```

Comment:
notatniki przetwarzające mają być oczyszczone z outputu.

Reply:
Jeżeli outputem tego notanika była wyświetlona tabela, to został on usunięty.